db/migrate/20191229204109_create_auth_app_configurations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAuthAppConfigurations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :auth_app_configurations do |t|
+      t.integer :user_id, null: false
+      t.string :encrypted_otp_secret_key, null: false
+      t.string :name, null: false
+      t.integer :totp_timestamp
+      t.timestamps
+      t.index [:user_id, :created_at], unique: true
+      t.index [:encrypted_otp_secret_key], unique: true
```

Comment:
not sure if this encrypted_otp_secret_key is needed

Reply:
Yep, no clue what it is indexed